Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM moyix/triton_with_ft:22.09
+
+# Install dependencies: torch
+RUN pip3 install -U torch --extra-index-url https://download.pytorch.org/whl/cu116
+RUN pip3 install -U transformers bitsandbytes accelerate
```

Comment:
`moyix/triton_with_ft:22.09` comes from this repo https://github.com/moyix/fastertransformer_backend/blob/main/docker/Dockerfile

Seems to me more logical to add the dependencies there?